nrf_modem/doc/sockets/socket_options_func.rst

```diff
@@ -59,6 +59,8 @@ The following table shows all socket options supported by the Modem library.
 +-----------------+---------------------------------+------------------------+------------+--------------------------------------------------------------------------------------------+
 | NRF_SOL_SOCKET  | NRF_SO_EXCEPTIONAL_DATA         | ``int``                | get/set    | Send data on socket as part of exceptional event.                                          |
 +-----------------+---------------------------------+------------------------+------------+--------------------------------------------------------------------------------------------+
+| NRF_SOL_SOCKET  | NRF_SO_KEEPOPEN                 | ``int``                | get/set    | Keep the socket open when its PDN connection is lost, or the device is put in flight mode. |
```

Comment:
This changes the meaning of the sentence, what I mean to say is that the socket will remain open when the PDN connection is lost, or in case the device is put in flight mode (which disables connectivity).

Reply:
okay, my bad, I misunderstood that the socket option does two actions here.
Then,  we could rephrase as : _Keep the socket open when its PDN connection is lost or the device is set to flight mode (``CFUN=4``)._

Reply:
Ok, I rephrased it but kept the `CFUN=4` out of here since it's only a short summary in a table; we mention in it in the description below.